modules/glusterfs/data/org.storaged.Storaged.glusterfs.xml

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE node PUBLIC
+"-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+
+<!--
+ Copyright (C) 2015 Samikshan Bairagya <sbairagy@redhat.com>
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General
+ Public License along with this library; if not, write to the
+ Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+ Boston, MA 02111-1307, USA.
+-->
+
+  <!-- ********************************************************************** -->
+
+  <!--
+      org.storaged.Storaged.Manager.GlusterFS:
+      @short_description: glusterd add-on to the manager singleton
+      @since: 2.4.1
+
+      Additional interface with glusterfs specifics for top-level manager
+      singleton object located at the object path
+      <literal>/org/storaged/Storaged/Manager</literal>.
+  -->
+  <interface name="org.storaged.Storaged.Manager.GlusterFS">
+    <!-- prereq: org.storaged.Storaged.Manager -->
+
+    <annotation name="org.gtk.GDBus.C.Name" value="Manager_GlusterFS"/>
+    <!--
+        Reload:
+
+        Reloads the gluster state in storaged
+    -->
+    <method name="Reload"/>
```

Comment:
Please, add the`options` argument and `@since` tag:

``` xml
<!--
        Reload:
        @options: Additional options.
        @since: 2.4.1

        Reloads the gluster state in storaged.

        No additional options are currently defined.
-->
<method name="Reload">                             
  <arg name="options" type="a{sv}" direction="in"/>
</method>                                          
```

UDisks2 D-Bus API uses this additional parameter for future changes; but mainly for *Polkit parameter(s) passing*, etc.

**NOTE:** This applies for all the methods introduced in this module.